Model/LogHandler.php

```diff
@@ -15,6 +15,12 @@ class LogHandler
     private $logFile;
     private $logDir;
     private $ecommerce;
+    private $reponse;
+    private $lockfile;
+    private $confdays;
+    private $confweight;
```

Comment:
this should be camel case lockFile, confDays, confWeight, according to other attributes nomeclature

Reply:
that utility is replaced in another PR